markdown/dev/reference/api/part/config/measurements/en.md

```diff
@@ -38,3 +38,12 @@ const part = {
 }
 ```
 
+<Tip>
+
+Although they are specified via the part configuration `optionalMeasurements`
+property, optional measurements are accessed via the 'measurements'
+settings property.
+
+(There is no `optionalMeasurements` settings property.)`
+
+</Tip>
```

Comment:
Sorry to nitpick, but I don't think this goes here. It's not relevant to a user who has come to the docs to figure out how to specify which measurements are required by a part, and I worry it's confusing to see in this context

I think it should move to `markdown/dev/api/part/draft/en.md`, where it is relevant to a user who is looking at how to access measurements in the draft method and there is currently no note